contrib/oci-seccomp-gen/parse_location.go

```diff
@@ -0,0 +1,49 @@
+package ociseccompgen
+
+import (
+	"os"
+	"os/user"
+	"strconv"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+)
+
+// DefaultFullPath returns the default full path/name for output configuration files
+func DefaultFullPath() string {
+	return (parseLocation(pwd(), parseNameWithNumber("manhattan", 0)))
```

Comment:
`parse_location.go` was used specifically for the `manhattan` tool so I can see dropping it.